app/controllers/educators_controller.rb

```diff
@@ -22,6 +22,20 @@ def names_for_dropdown
     render json: filtered_names(params[:term], school)
   end
 
+  def notes_feed
+    time_interval = Date.today - 2400
+    notes = EventNote.where(educator_id: current_educator.id).where("recorded_at >= ?", time_interval).order("recorded_at DESC")
+    student_ids = notes.map {|note| note.student_id}
+    student_ids.uniq!
+    students = Student.find(student_ids)
```

Comment:
@alexsoble Is there somewhere in the code base where we do something similar? This seems pretty inefficient, not to mention it doesn't associate a student to each note, which is what I ultimately need to do. I have all tomorrow morning to work on SI stuff so I am sure I can come up with something, but if something already exists and you know where it is, that would be useful. Thanks!

Reply:
@gabivoicu When you mention associating each student to a note, that makes me thing we'll want to use something like `#includes` or `#join` to avoid an N+1 query problem.

Maybe something like: 

```
EventNote.includes(:student)
  .where(educator_id: current_educator.id)
  .where("recorded_at >= ?", time_interval)
  .order("recorded_at DESC")
  ...
```